collections/_release_4_4/entry-highlights-initial-android-editor-support-for-XR-devices.md

```diff
@@ -0,0 +1,21 @@
+---
+type: entry
+section: highlights
+rank: 2
+importance: 1
+anchor: "initial-android-editor-support-for-XR-devices"
+title: "Initial Android editor support for XR devices"
```

Comment:
I guess it is "initial" because this is the first version of Godot with it, but, I dunno, lots of things here are new Godot features and don't say "initial". Let's just confidently say that it is :-)

```suggestion
anchor: "android-editor-support-for-XR-devices"
title: "Android editor support for XR devices"
```